### PR TITLE
Add Linux support

### PR DESCRIPTION
**Pull request by santigarcor**
Added an if statement that checks if the script is being run on OSX or Linux.

### CODE REVIEW

**Comment by mattstauffer** [comment]
Thanks for this PR. However lambo relies on Valet, which is mac only, so this seems unnecessary/unhelpful. But am I missing something?
Thanks!

**Comment by santigarcor** [comment]
@mattstauffer well [this](https://github.com/cpriego/valet-ubuntu) fork of valet supports Linux. Indeed currently i'm using valet on Ubuntu. So i think this PR could be useful for people using that fork

**Comment by mattstauffer** [comment]
@santigarcor wow, had no idea this exists.

I don't have an Ubuntu machine to test on; will have to ask for help in reviewing it.

**Comment by santigarcor** [comment]
@mattstauffer if you want i can make a video showing how lambo works in Ubuntu and then send it to you

File: lambo [hunk 1]
```diff
@@ -117,12 +117,21 @@ fi
...
 # Update .env to point to this database with `root` username and blank pw,
 # like Mac MySQL defaults, and appropriate domain
-sed -i '' "/DB_DATABASE/s/homestead/$PROJECTNAME/" .env
-sed -i '' '/DB_USERNAME/s/homestead/root/' .env
-sed -i '' '/DB_PASSWORD/s/secret//' .env
-sed -i '' "/APP_URL/s/localhost/$PROJECTNAME\.$TLD/" .env
-
-open "http://$PROJECTNAME.$TLD"
+if [ "$(uname)" == "Darwin" ]; then
+    sed -i '' "/DB_DATABASE/s/homestead/$PROJECTNAME/" .env
```
Comment (mattstauffer) [inline lambo]: Hm, I wonder if there's a way to put these strings in an array and iterate over them so they're not all duplicated?

**Comment by mattstauffer** [comment]
@santigarcor I had a friend test it--I can see it works in Ubuntu, thanks! Now just want to make sure the structure is good. :)

File: lambo (hunk 1 above)
Comment (mattstauffer) [inline lambo]: Maybe we could do it something like this? http://www.cyberciti.biz/faq/bash-for-loop-array/

It looks like the strings are the same, right? If so, then we can store them in an array and just let the command be different each time

**Comment by santigarcor** [comment]
@mattstauffer no, the strings are different in an empty string

**Comment by mattstauffer** [comment]
@santigarcor not sure what you mean. Here's what I'm imagining, in pseudo code:

```
commands = [
    "/DB_DATABASE/s/homestead/$PROJECTNAME/",
    '/DB_USERNAME/s/homestead/root/',
    '/DB_PASSWORD/s/secret//',
    "/APP_URL/s/localhost/$PROJECTNAME\.$TLD/"
]

if (mac)
    foreach(commands as command)
        sed -i '' command .env
    endforeach
else if (linux)
    foreach(commands as command)
        sed -i command .env
    endforeach
endif
```

**Comment by santigarcor** [comment]
@mattstauffer never mind just forget what i said.

If you want i can try to make something like that an commit into this PR

**Comment by mattstauffer** [comment]
@santigarcor if you feel up for it, that'd be awesome! If not, let me know and i'll do it

**Comment by santigarcor** [comment]
@mattstauffer i'll do it :smile:

**Comment by santigarcor** [comment]
@mattstauffer Done

**Comment by mattstauffer** [comment]
Fantastic--thanks @santigarcor!

**Pull request merged**